README.md

```diff
@@ -277,7 +277,7 @@ Commands:
     Emit a script that will export environment variables.
 
     -p, --profile=PROFILE      The AWS profile to save the temporary credentials. (env: SAML2AWS_PROFILE)
-        --shell=bash           Type of shell environment. Options include: bash, powershell, fish, env
+        --shell=bash           Type of shell environment. Options include: bash, /bin/sh, powershell, fish, env
```

Comment:
I don't use this part of `saml2aws` often, but just curious why the option would be named `/bin/sh` instead of `sh`. The other shell types don't take full paths.

Reply:
To ensure that system shell is invoked (regardless of PATH configuration)

I will move to `sh` if you think that is better

I didn't propose renaming /bin/bash and /bin/fish, is it might be breaking existing configs.

Reply:
ok yeah I can see how that works from the invoking perspective; but the `script` subcommand does not actually invoke the shell it generates the variables for, no? I guess it's used for the shebang (though I can't see where it's used in this PR).

I might also argue that we _should_ respect the `PATH` variable, rather than deciding for the end user/sysadmin not to use what they configured.

Also looking at it from an end-user point of view, I think it's much nicer to refer to it as `sh`, both for consistency and just in general (I would not want to be expected to give the full path to a powershell executable for example, especially when it's not really needed to run this subcommand).

Reply:
ah I see this has already been merged... if it has not been released, I would strongly urge reconsidering to use `sh` instead of the full path.

Perhaps, an additional option could be added to the `script` subcommand to provide a full path to the shell in question (the short name determines template, while the full path determines the binary).

This would also be nice for the `powershell` template since the binary can be named `powershell.exe`, `pwsh.exe`, or `pwsh`, for example.

Reply:
@gliptak @mapkon pinging just because this conversation is marked "resolved", just didn't want the comments to get lost